CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Node.js Buildpack Changelog
 
 ## main
+- Update node version to 14 ([#885](https://github.com/heroku/heroku-buildpack-nodejs/pull/885))
+
+# v182 (2020-01-05)
 - add Node 14.15.3 and 15.5.0 to inventory ([#881](https://github.com/heroku/heroku-buildpack-nodejs/pull/881))
 - Report on parsing errors for `resolve-version.rs` ([#883](https://github.com/heroku/heroku-buildpack-nodejs/pull/883))
 - Add an environment variable for using npm install (instead of npm ci) ([#882](https://github.com/heroku/heroku-buildpack-nodejs/pull/882))
+- add Node 10.23.1, 12.20.1, 14.15.4, and 15.5.1 to inventory ([#886](https://github.com/heroku/heroku-buildpack-nodejs/pull/886))
+- Update node version to 14 ([#885](https://github.com/heroku/heroku-buildpack-nodejs/pull/885))
```

Comment:
It looks like this still needs to be rebased since there's extra additions to the change log? I'm not sure how this ended up like this, but maybe rollback your changes to an earlier commit. 😅